### PR TITLE
feat(wallet): Network Identifications in Portfolio

### PR DESCRIPTION
**Pull request by Douglashdaniel**
## Description 
Added Network Identifications in Portfolio

1) Now displays a `Network` icon on the `Portfolio Asset Item` along with a `Description` of the network for an asset.
2) Now displays a `Network` icon for each network when selecting an `Network` unless there is now icon url specified for that network, will then default to the placeholder orb.

Resolves <https://github.com/brave/brave-browser/issues/20902>

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

https://user-images.githubusercontent.com/40611140/152903292-b9dc8f1c-e64f-4962-8156-669a65023f25.mov

### CODE REVIEW

File: components/brave_wallet_ui/components/shared/create-network-icon/index.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+} from '../../../assets/asset-icons'
+
+interface Props {
+  network: BraveWallet.EthereumChain
+  marginRight: number
+}
+
+function CreateNetworkIcon (props: Props) {
+  const { network, marginRight } = props
+  console.log(network)
```
Comment (onyb) [inline components/brave_wallet_ui/components/shared/create-network-icon/index.tsx:17]: remove me

File: components/brave_wallet_ui/components/shared/create-network-icon/index.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    if (isStorybook) {
+      return true
+    }
+    return false
+  }, [isRemoteURL, isDataURL, networkImageURL])
+
+  // Will remove these hardcoded chainId's.
+  // We need to return a normal Ethereum Icon URL for Ethereum Mainnet
+  // and return a grayed out Ethereum Icon Url for Ethereum Test Networks from the backend.
+  const testNetworkList = ['0x4', '0x3', '0x5', '0x2a', '0x539']
```
Comment (onyb) [inline components/brave_wallet_ui/components/shared/create-network-icon/index.tsx:40]: Good enough for now, but ideally we want to have a `isMainnet` or `isTestnet` field in `EthereumChain` in the future.

**Review by onyb — APPROVED** [review]
Great stuff! 🚀

File: components/brave_wallet_ui/components/shared/create-network-icon/index.tsx (hunk 1 above)
Comment (Douglashdaniel) [reply components/brave_wallet_ui/components/shared/create-network-icon/index.tsx]: lol removed!

File: components/brave_wallet_ui/components/shared/create-network-icon/index.tsx (hunk 2 above)
Comment (Douglashdaniel) [reply components/brave_wallet_ui/components/shared/create-network-icon/index.tsx:39]: Yes! I agree!

**Pull request merged**